packages/app/src/systems/Transaction/services/transaction.tsx

```diff
@@ -251,9 +247,14 @@ export class TxService {
       txSummary.isStatusFailure = true;
       txSummary.status = TransactionStatus.failure;
 
+      // Fallback to the values from the transactionRequest
+      if ('gasLimit' in transactionRequest) {
+        txSummary.gasUsed = transactionRequest.gasLimit;
+      }
```

Comment:
This will make `FeeOptions` always visible. Even with simulation error 
FYI @LuizAsFight